Backend/Remora.Discord.API.Abstractions/API/Objects/Interactions/ApplicationCommands/IBulkApplicationCommandData.cs

```diff
@@ -52,4 +52,10 @@ public interface IBulkApplicationCommandData
 
     /// <inheritdoc cref="IApplicationCommand.DescriptionLocalizations"/>
     Optional<IReadOnlyDictionary<string, string>?> DescriptionLocalizations { get; }
+
+    /// <inheritdoc cref="IApplicationCommand.DefaultMemberPermissions"/>
+    Optional<IDiscordPermissionSet?> DefaultMemberPermissions { get; }
+
+    /// <inheritdoc cref="IApplicationCommand.DmPermission"/>
+    Optional<bool> DmPermission { get; }
```

Comment:
Remember to correct the nullability here as well, and remove `DefaultPermission`.

Reply:
``DMPermission`` here too?